Configuration/PyReleaseValidation/python/relval_steps.py

```diff
@@ -1162,6 +1164,7 @@ def lhegensim(fragment,howMuch):
 steps['RECODR2_25nsreHLT']=merge([{'--hltProcess':'reHLT'},steps['RECODR2_25ns']])
 steps['RECODR2_50nsreHLT']=merge([{'--hltProcess':'reHLT'},steps['RECODR2_50ns']])
 steps['RECODR2_2016reHLT']=merge([{'--hltProcess':'reHLT','--conditions':'auto:run2_data_relval'},steps['RECODR2_2016']])
+steps['RECODR2newL1repack_2016reHLT']=merge([{'-s':'L1REPACK:FullSimTP,RAW2DIGI,L1Reco,RECO,EI,PAT,ALCA:SiStripCalZeroBias+SiStripCalMinBias+TkAlMinBias+EcalESAlign,DQM:@standardDQM+@miniAODDQM','--hltProcess':'reHLT'},steps['RECODR2_2016']])
```

Comment:
@rekovic:
```
cmsrel CMSSW_8_1_X_2016-07-12-2300
cd CMSSW_8_1_X_2016-07-12-2300/src/
cmsenv
git cms-merge-topic 15182
scramv1 b -j 8
runTheMatrix.py -nel 136.7321
```
gives me:
```
 [1]: input from: /ZeroBias/Run2016B-v2/RAW with run [] 
 [2]: cmsDriver.py step2  --datatier FEVTDEBUGHLT --conditions auto:run2_hlt_relval -s L1REPACK:FullSimTP,HLT:@relval2016 --process reHLT --data  --era Run2_2016 --eventcontent FEVTDEBUGHLT -n 100 
 [3]: cmsDriver.py step3  --conditions auto:run2_data_relval -s L1REPACK:FullSimTP,RAW2DIGI,L1Reco,RECO,EI,PAT,ALCA:SiStripCalZeroBias+SiStripCalMinBias+TkAlMinBias+EcalESAlign,DQM:@standardDQM+@miniAODDQM --runUnscheduled  --process reRECO --data  --era Run2_2016 --eventcontent RECO,MINIAOD,DQM --hltProcess reHLT --scenario pp --datatier RECO,MINIAOD,DQMIO --customise Configuration/DataProcessing/RecoTLR.customisePostEra_Run2_2016 -n 100 
 [4]: cmsDriver.py step4  --data  --filetype DQM --conditions auto:run2_data_relval -s HARVESTING:@standardDQM+@miniAODDQM --scenario pp -n 100 
```
just to be clear, the worry is that the `L1REPACK:FullSimTP` in step2 and step3 are consuming different Global Tags?